### PR TITLE
Backfill HESA codes for no disabilities

### PR DESCRIPTION
**Pull request by steventux**
## Context

Depends on https://github.com/DFE-Digital/apply-for-teacher-training/pull/4921

Application forms which have empty equality and diversity `disabilities` array fields should present the HESA `00` code, backfill this data as it was not being correctly populated by `DisabilityStatusForm`. ([See dependency PR](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4921) for fix)

## Changes proposed in this pull request

Adds a data migration to backfill any application forms which have empty equality and diversity `disabilities` and `hesa_disabilities` arrays, as this indicates they have been set by the E&D questionnaire but the `00` no disabilities hesa code was not correctly populated.

## Link to Trello card

https://trello.com/c/cQurakCV/3772-tribal-select-no-disability-returns-blank-should-be-00

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/services/data_migrations/backfill_none_hesa_disabilities_codes.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module DataMigrations
+  class BackfillNoneHesaDisabilitiesCodes
+    TIMESTAMP = 20210610111514
+    MANUAL_RUN = false
+
+    def change
+      no_disabilities_hesa_code = Hesa::Disability.find('no').hesa_code
+
+      ApplicationForm
```
Comment (cwrw) [inline app/services/data_migrations/backfill_none_hesa_disabilities_codes.rb:9]: Do we want to avoid the  touching the records, also not having audits?

Comment (cwrw) [reply app/services/data_migrations/backfill_none_hesa_disabilities_codes.rb:9]: had a think around this, and actually its best if we do, as we are updating the HESA code so good for integrations to pull the changes in. I'll check how many records this affects though

**Review by cwrw — APPROVED** [review]
👍

**Comment by steventux** [comment]
@cwrw I've amended with a `NULL` clause https://github.com/DFE-Digital/apply-for-teacher-training/pull/4930/commits/54201fe564f90b4402d1215f25f956ca09136c17#diff-8f62d42639088865b9bdc0c963e057eb59c95c8e231240a12dbc4aca4b0da22aR11 and test

**Review by cwrw — APPROVED** [review]

**Pull request merged**